Blog post for the contest results (#458)
haiku/website: content/news/2020-12-07-contest-results.md

4. 4th place: 2 points
5. 5th place: 1 point

According to votes, final score were calculated as below:

| **Theme name**                    |**Points**|
|:------n----------------------------------|--:|
|**1.** burg's-sparkly-tones               |249|
|**2.** Komorebi                           | 50|
|**3.** Basics                             | 43|
|**4.** Zen                                | 42|
|**5.** Clack                              | 37|
|**6.** Yanqim                             | 25|
|**7.** Leafy                              | 23|
|**8.** Forest Dreams                      | 23|
|**9.** Low Modularian                     | 18|
|**10.** Zen Peace                         | 16|
|**11.** Meditation Time                   | 15|
|**12.** Bassling                          | 15|
|**13.** Straight Outta Cronton            | 13|
|**14.** Nature                            | 12|
|**15.** Supposedly Not Too Annoying Theme | 10|
|**16.** The Poet Operating System Theme   |  7|
|**17.** Daniel Diaz                       |  6|

You can download the votes' breakdown table from [this link](/files/sound-contest/sound_contest_cotes.ods) (.ods file, 20 KiB).

## What next?

> Winner entry will be used as the default sound theme in Haiku R1. Expect to see this theme in action in the upcoming beta releases. You can still use any sound theme you please via setting them up manually in "Sounds" preferences. It will also be possible to install and use the other entries via HaikuDepot, once the underlying foundation for this feature is implemented.

Reviewer: Let's not promise this for the next beta. "Sounds" prefs need to support themes, or at the least muting sounds. I'm pretty sure we'll decide to have all sounds (besides startup and beep) muted by default.

```suggestion
The winner entry will be used as the default sound theme in Haiku R1 once proper theming is implemented in the "Sounds" preferences. You can still use any sound theme you please (or none) via setting them up manually in the "Sounds" preferences. It will also be possible to install and use the other entries via HaikuDepot, once the underlying foundation for this feature is implemented.
```

Author: I think we can already use the winner theme by default, no? Profiling is only needed if the other sound themes were to be added.

Reviewer: Yes. But I would hate to have all those sound events active by default and having to clear all of them manually.

Author: It makes sense, but the last time I checked, the improvements ticket which included the mock-up for profiling was not under R1 milestone. If no one implements profiling, the default theme will probably be included anyway I guess.

Reviewer: You mean, included, but not pre-configured in the Sounds prefs?

Author: I think they would be pre-configured... By BeOS' logic, someone has to manually replace the sounds anyway if they want to customise them.

Reviewer: As I said, I think most people would hate to boot into a newly installed Haiku with every mouse click, key type and window action beep-beep-booping until I manually set those dozen events manually to <none>.
As we don't know how it will be done eventually, we should phrase it more fuzzy. How about:

```suggestion
The winning sound theme will be included in Haiku R1. You can still use any sound theme you please via setting them up manually in "Sounds" preferences or install and use the other entries via HaikuDepot, once the underlying foundation for this feature is implemented.
```

Author: Updated. I don't know about profiling, but on/off checkboxes should definitely be implemented before R1, and most of them should be off by default.

Reviewer: +1

Author: In case it's not implemented, we can just include the prominent ones, like notifications, startup etc.
